src/briefcase/input_wrapper.py

```diff
@@ -0,0 +1,74 @@
+
+
+class InputWrapper:
```

Comment:
Minor code organization thing; I think this class belongs in console.py, rather than being in a standalone file.